src/Adapter/LinkAdapter.php

```diff
@@ -58,7 +58,13 @@ public function __construct(\Link $link = null)
     public function getAdminLink($controller, $withToken = true, $sfRouteParams = [], $params = [])
     {
         if ((new ShopContext())->isShop17()) {
-            return $this->link->getAdminLink($controller, $withToken, $sfRouteParams, $params);
+            $adminLink = $this->link->getAdminLink($controller, $withToken, $sfRouteParams, $params);
+            // We have problems with links in our zoid application, since some links generated don't have domain they redirect to CDN domain
+            if (!str_contains($adminLink, 'http')) {
```

Comment:
```suggestion
            if (false === strpos($adminLink, 'http')) {
```
`str_contains` is PHP 8 only